cpp/include/cudf/io/orc.hpp

```diff
@@ -346,12 +346,9 @@ class orc_reader_options_builder {
  *
  * The following code snippet demonstrates how to read a dataset from a file:
  * @code
- *  ...
- *  std::string filepath = "dataset.orc";
- *  cudf::orc_reader_options options =
- * cudf::orc_reader_options::builder(cudf::source_info(filepath));
- *  ...
- *  auto result = cudf::read_orc(options);
```

Comment:
other documentation string in this file still has `cudf::orc` usage. 
`cudf::write_`, `cudf::read_` also. Check all the edited files in this PR.